dedupe-API.md

```diff
@@ -0,0 +1,30 @@
+
+
+# API
+
+## `/match`
+
+Returns the CJ identifier that the dedupe service believes matches a HMIS identifier (if any).
+
+### Parameters
+- `hmis`: a valid hmis id
+
```

Comment:
Will we be able to do the opposite for passing in a cj id and getting back a matching hmis id and score? or will that be a different endpoint?

Reply:
Do you need both endpoints?

Reply:
Yes. Is that within your current scope of work?